tests/unit/test_data.py

```diff
@@ -99,8 +102,8 @@ def test_atleast_2d_consistent(arrays: tuple[Array]) -> None:
 @pytest.mark.parametrize(
     "data_type",
     [
-        partial(coreax.data.Data, DATA_ARRAY),
-        partial(coreax.data.SupervisedData, DATA_ARRAY, SUPERVISION),
+        jtu.Partial(coreax.data.Data, DATA_ARRAY),
+        jtu.Partial(coreax.data.SupervisedData, DATA_ARRAY, SUPERVISION),
```

Comment:
Happy for this to be out of scope, but it might be nice to wrap these in a `pytest.param` each to give them a friendly name for the logs.